### PR TITLE
Molecule variables - remove relative path

### PR DESCRIPTION
**Pull request by domenicbove**
# Description

- With this PR: https://github.com/confluentinc/cp-ansible/pull/701 , many of the variables using relative paths had to be changed. This PR removes the relative path logic and makes use of the `MOLECULE_SCENARIO_DIRECTORY` - simplifying things.
- Removed duplicating converge: stanzas from all the molecule.ymls, will make it easier in the future to change the converge playbook if need be!

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Ran many molecule scenarios manually, jenkins also will run

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml [hunk 1]
```diff
@@ -206,9 +204,9 @@ provisioner:
           sasl_protocol: plain
 
         kafka_connect_replicator_consumer_bootstrap_servers: mds-kafka-broker1:9092
-        kafka_connect_replicator_consumer_ssl_ca_cert_path: "molecule/{{scenario_name}}/generated_ssl_files/ca.crt"
-        kafka_connect_replicator_consumer_ssl_cert_path: "molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
-        kafka_connect_replicator_consumer_ssl_key_path: "rmolecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-key.pem"
+        kafka_connect_replicator_consumer_ssl_ca_cert_path: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/ca.crt"
+        kafka_connect_replicator_consumer_ssl_cert_path: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
+        kafka_connect_replicator_consumer_ssl_key_path: "r{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/{{inventory_hostname}}-key.pem"
```
Comment (domenicbove) [inline molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml:209]:
```suggestion
        kafka_connect_replicator_consumer_ssl_key_path: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/{{inventory_hostname}}-key.pem"
```

File: molecule/ccloud/prepare.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@
 
     - name: Copy CA Cert to Host
       copy:
-        src: "../../../../generated_ssl_files/{{ssl_self_signed_ca_cert_filepath|basename}}"
+        src: "../../generated_ssl_files/{{ssl_self_signed_ca_cert_filepath|basename}}"
```
Comment (erikgb) [inline molecule/ccloud/prepare.yml:44]: What has happened here? Has the file been moved?

Comment (domenicbove) [reply molecule/ccloud/prepare.yml:44]: this is a bug left behind from the molecule dir move PR, it went down two directories.
i consider this a one off

**Review by erikgb — APPROVED** [review]
LGTM. If the build is green we are GTM.

**Pull request merged**